spec/translator.actions.lsl

```diff
@@ -32,35 +31,16 @@ define action SYMBOLIC_ARRAY (
     ): array<any>;
 
 
+
+/// specification development -related aspects
+
 define action DEBUG_DO (
         code: string // literal!
     ): any;
 
 
-// note: result variable should be passed EXPLICITLY!
-// usage example: action LOOP_FOR(i, 0, 10, +1, loop_body_proc(i, list, x, y));
-define action LOOP_FOR (
-        iterator: int32,   // variable!
-        lowerBound: int32,
-        upperBound: int32,
-        step: int32,
-        bodyProc: void     // subroutine call!
-    ): void;
-
-
-// note: result variable should be passed EXPLICITLY!
-// usage example: action LOOP_WHILE(a < b, loop_body_proc(a));
-define action LOOP_WHILE (
-        predicate: bool,
-        bodyProc: void   // subroutine call!
-    ): void;
-
-
-define action LOOP_BREAK (): void;
-
-
-
-/// specification development -related aspects
+define action GUID_NEXT (
```

Comment:
I don't understand this action... Can you explain it's purpose for me ? Or write a little example ?

Reply:
Added a comment with a note on how it should behave.